examples/langgraph-memanto/.env.example

```
@@ -0,0 +1,8 @@
+# Required: get a free key at https://console.moorcheh.ai/api-keys
+MOORCHEH_API_KEY=your_moorcheh_api_key_here
+
+# Optional demo controls
+MEMANTO_AGENT_ID=langgraph-support-demo
+MEMANTO_DEMO_USER_ID=maya-rivera
+MEMANTO_DEMO_MESSAGE=Can you remind me what we were working on yesterday and how I prefer updates?
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MEMANTO_DEMO_MESSAGE key should go before the MEMANTO_DEMO_USER_ID key

(UnorderedKey)